constellations/clients/orionFakeDataFiller/filler.go

```diff
@@ -228,28 +238,123 @@ func runFiller(hostAddress string) {
 	)
 
 	// Create sessions
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_1",
+		"ap_java_2020_fall_class1",
 		"false",
+		3,
 	)
 
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_2",
+		"ap_calculus_2020_fall_class1",
 		"true",
+		2,
+	)
+
+	// Create Accounts
+	createAccount(
+		hostAddress,
+		"emailaddress1@gmail.com",
+		"jhdgjhddjhdjuj",
+	)
+	createAccount(hostAddress,
+		"emailaddress2@gmail.com",
+		"2redssssa",
+	)
+
+	// Create Users
+	createUser(
+		hostAddress,
+		"Joe",
+		"Smith",
+		"",
+		"emailaddress1@gmail.com",
+		"301-123-4567",
+		false,
+		1,
+		"notes1",
+		"schoolone",
+		2001,
+	)
+
+	createUser(
+		hostAddress,
+		"Joe",
+		"Smith",
+		"Mom",
+		"emailaddress1@gmail.com",
+		"301-123-4567",
+		true,
+		1,
+		"notes1",
+		"",
+		0,
+	)
+
+	createUser(
+		hostAddress,
+		"Billy",
+		"Bob",
+		"Joe",
+		"emailaddress2@gmail.com",
+		"301-123-4568",
+		false,
+		2,
+		"notes2",
+		"schooltwo",
```

Comment:
I think this will also be useful if it was a "real" high school name.
`Montgomery Richard HS` or something so when we display on the UI (front-end website), it looks closer to what we expect. `schooltwo` seems like it's an id or something